### PR TITLE
Support specifying externalTrafficPolicy in Services created by listener-operator

### PR DESCRIPTION
**Pull request by sbernauer**
# Description

Needs https://github.com/stackabletech/operator-rs/pull/562

## Review Checklist

- [ ] Code contains useful comments
- [ ] CRD change approved (or not applicable)
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
- [ ] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -6,5 +6,5 @@ members = ["rust/operator-binary", "rust/csi-grpc"]
 h2 = { git = "https://github.com/stackabletech/h2.git", branch = "feature/grpc-uds" }
 
 [patch."https://github.com/stackabletech/operator-rs.git"]
-# stackable-operator = { git = "https://github.com/stackabletech//operator-rs.git", branch = "feature/listener-implied-labels" }
+stackable-operator = { git = "https://github.com/stackabletech//operator-rs.git", branch = "feat/listener-service-external-traffic-policy" }
```
Comment (sbernauer) [inline Cargo.toml:9]: Reminder

Comment (nightkr) [reply Cargo.toml:9]: @sbernauer This is unblocked now that we have https://github.com/stackabletech/operator-rs/pull/773 right?

Comment (sbernauer) [reply Cargo.toml:9]: Oh yes, I will pick this PR up again (feel free to push and merge yourself if I'm too slow)

Comment (nightkr) [reply Cargo.toml:9]: I'm not in a rush here, just ran into the PR by accident. ^^

**Pull request closed without merging**